packages/webdriver-utils/src/metadata/commonMobileMetaDetaResolver.js

```diff
@@ -0,0 +1,42 @@
+export default class CommonMobileMetaDataResolver {
+  constructor(driver, opts = {}) {
+    this.driver = driver;
+    this.capabilities = opts;
+  }
+
+  browserName() {
+    return this.capabilities.browserName.toLowerCase();
+  }
+
+  osName() {
+    let osName = this.capabilities.os.toLowerCase();
+    if (osName === 'mac' && this.browserName() === 'iphone') {
+      osName = 'ios';
+    }
```

Comment:
ideally create a metadata subclass here. Dont add `if` [ in long run you will have so many such conditions, polluting the code and making it unreadable ]

Reply:
Once we add more roles over here, will split it into another subcalls.
Keeping it as is for now.